easy_access/db/base.py

```
@@ -33,9 +35,24 @@ async def ensure_db_inited(settings: Settings | None = None) -> bool | None:
-        return None
+    # Check if engine actually exists and is not disposed
+    engine = get_engine()
+    if _DB_INITIALIZED and engine is not None:
+        # Additional check: try to verify engine is usable
+        try:
+            # Simple check - if we can get the pool, engine should be OK
+            if hasattr(engine, "pool") and engine.pool is not None:
+                return None
+        except Exception:
+            # If engine check fails, mark as needing re-init
+            pass
```

Ruff (0.14.6)

`try`-`except`-`pass` detected, consider logging the exception

(S110)